docs/user-guide/menu-bar.md

```diff
@@ -142,47 +138,51 @@ When all the properties have been set, selecting the <img src="../img/button/sav
 
 Once a bookmark has been created, it is always possible to *Edit* it <img src="../img/button/edit-service-icon.jpg" class="ms-docbutton"/> or *Remove* it <img src="../img/button/delete-service-icon.jpg" class="ms-docbutton"/> from the list.
 
-## Burger Menu
+## Options Bar
 
-The *Burger Menu* is an important tools container that allows the user to perform different operations and take a look at several information:
+The *Options Bar* is an important tools bar, positioned on the right side of the map viewer, that allows the user to perform different operations and take a look at several information:
```

Comment:
The *Side Toolbar* is an important component, positioned on the right side of the map viewer, that provides to the user the access to different tools of MapStore. The following tools are the ones available by default: